cellmincer/denoise/argparse.py

```diff
@@ -51,30 +52,30 @@ def add_subparser_args(subparsers: argparse) -> argparse:
         dest='model_type',
         default=None,
         required=True,
-        help='Name of model class.')
+        help='Name of model class. Options: "spatial-unet-2d-temporal-denoiser"')
```

Comment:
Since we support a single model type at the moment, I suggest hiding "type" from argparse and injecting `"spatial-unet-2d-temporal-denoiser"` at an appropriate place downstream.